cluster/mesos/docker/docker-compose.yml

```diff
@@ -157,10 +157,12 @@ scheduler:
   volumes:
   - ./static-pod.json:/opt/static-pods/static-pod.json
 keygen:
-  image: mesosphere/kubernetes-mesos-keygen
+  image: mesosphere/kubernetes-keygen:v1.0.0
```

Comment:
About the different keygen calls: we had the discussion before I think. Let's leave it like it is. Not worth the effort to change it now.